keyboards/dztech/dz60/readme.md

```diff
@@ -10,8 +10,14 @@
 * Hardware Supported: DZ60 60% Mechanical Keyboard PCB
 * Hardware Availability: [KBDfans](https://kbdfans.com/collections/60/products/dz60-60-pcb)
 
+There are two versions of the DZ60RGB-ANSI. Please use the appropriate firmware for your board.
```

Comment:
```suggestion
There are two versions of the DZ60 PCB. Please use the appropriate firmware for your board.
```

You may also add a note that DZ60RGB is a completely different board… but that probably won't help anyway.